ivy/functional/backends/jax/device.py

```diff
@@ -76,6 +66,10 @@ def _to_device(x, device=None):
     return x
 
 
+# --- Main --- #
```

Comment:
we should rename this section to `Extra` if it isn't a part of the Array API Standard